src/python/pants/backend/awslambda/python/rules.py

```diff
@@ -31,30 +30,25 @@
 )
 from pants.engine.fs import Digest, MergeDigests
 from pants.engine.process import ProcessResult
-from pants.engine.rules import Get, collect_rules, rule
+from pants.engine.rules import Get, MultiGet, collect_rules, rule
 from pants.engine.unions import UnionRule
 from pants.option.global_options import GlobalOptions
 from pants.util.logging import LogLevel
 
 
 @dataclass(frozen=True)
-class PythonAwsLambdaFieldSet(PackageFieldSet, AWSLambdaFieldSet):
+class PythonAwsLambdaFieldSet(PackageFieldSet):
     required_fields = (PythonAwsLambdaHandler, PythonAwsLambdaRuntime)
 
     handler: PythonAwsLambdaHandler
     runtime: PythonAwsLambdaRuntime
     output_path: OutputPathField
 
 
-@dataclass(frozen=True)
-class LambdexSetup:
-    requirements_pex: Pex
```

Comment:
There was no need for a dedicated rule to set up lamdex, and it was likely resulting in less concurrency than we wanted.